x-pack/plugins/lens/public/visualizations/partition/toolbar.tsx

```diff
@@ -346,25 +346,29 @@ export function DimensionDataExtraEditor(
 
   return (
     <>
-      <CollapseSetting
-        value={currentLayer?.collapseFns?.[props.accessor] || ''}
-        onChange={(collapseFn) => {
-          props.setState({
-            ...props.state,
-            layers: props.state.layers.map((layer) =>
-              layer.layerId !== props.layerId
-                ? layer
-                : {
-                    ...layer,
-                    collapseFns: {
-                      ...layer.collapseFns,
-                      [props.accessor]: collapseFn,
-                    },
-                  }
-            ),
-          });
-        }}
-      />
+      {![...currentLayer.primaryGroups, ...(currentLayer.secondaryGroups ?? [])].includes(
```

Comment:
@andrewctate It seems like this doesn't work - "collapse by" is not shown:
<img width="352" alt="Screenshot 2022-11-10 at 16 26 31" src="https://user-images.githubusercontent.com/1508364/201136159-b82e25da-f0e7-4ffe-b98c-17f42c9c8d51.png">

Reply:
Good catch. Looks like I had the logic flipped. Fixed now.